### ISSUE TITLE
About the loss

### ISSUE THREAD

**Issue by wnma3mz** [ngailapdi/LWF#1]
Thank you very much for your project, which has helped me a lot. But I have some questions about the definition of loss. [line 152](https://github.com/ngailapdi/LWF/blob/baa07ee322d4b2f93a28eba092ad37379f565aca/model.py#L152)

Why do I need to recopy the original model and `forward` it. May I directly copy the value of `logits` of [line 149](https://github.com/ngailapdi/LWF/blob/baa07ee322d4b2f93a28eba092ad37379f565aca/model.py#L149). Or, what are the advantages or disadvantages of doing this?

Maybe I have overlooked some parts of the paper and hope to get your help. Any suggestions may be of great help to me, thank you very much!

**Comment by wnma3mz** [comment]
Sorry to bother you, I have found the answer in the paper of Fig3.
